docs/using/common.md

```diff
@@ -165,10 +165,13 @@ The certificate file or PEM may contain one or more certificates (e.g., server,
 
 For additional information about using SSL, see the following:
 
-- The [docker-stacks/examples](https://github.com/jupyter/docker-stacks/tree/master/examples) for information about how to use
+- The [docker-stacks/examples](https://github.com/jupyter/docker-stacks/tree/master/examples)
+  for information about how to use
   [Let's Encrypt](https://letsencrypt.org/) certificates when you run these stacks on a publicly visible domain.
-- The [jupyter_notebook_config.py](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_notebook_config.py) file for how this Docker image generates a self-signed certificate.
-- The [Jupyter Notebook documentation](https://jupyter-notebook.readthedocs.io/en/latest/public_server.html#securing-a-notebook-server) for best practices about securing a public notebook server in general.
+- The [`jupyter_server_config.py`](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_server_config.py)
+  file for how this Docker image generates a self-signed certificate.
```

Comment:
Thanks, I like this style of line breaks in the docs as well 👍

Reply:
😄